### PR TITLE
Homepage 

### PR DESCRIPTION
**Pull request by surbhi84**
Home and video listing page is the same which contains of a sidebar and navbar.
The huge number of file changes consists of svgs and a minor change I made throughout the project, please ignore that.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+        <Route path="/" element={<NestedRoutes />}>
+          <Route path="/" element={<Home />} />
+          <Route path="/trending" element={<Trending />} />
+          <Route path="/history" element={<History />} />
+          <Route path="/myplaylist" element={<MyPlaylist />} />
+          <Route path="/watchlater" element={<WatchLater />} />
+          <Route path="/likedvideos" element={<LikedVids />} />
+          <Route path="/settings" element={<Settings />} />
+          <Route path="/video/:id" element={<FullScreen />} />
+          <Route path="/ts" element={<Mockman />} />
```
Comment (aurobindoGupta) [inline src/App.js:30]: what is /ts

File: src/backend/db/videos.js [hunk 2]
```diff
@@ -5,17 +5,110 @@
 
 export const videos = [
   {
-    _id: "Wo5dMEP_BbI",
-    title: "Awesome Video about Coding",
-    description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
-    creator: "Soham Shah",
+    id: "OwxU-qdyCUM",
```
Comment (aurobindoGupta) [inline src/backend/db/videos.js:8]: we can use uuid() here too

File: src/backend/db/videos.js [hunk 3]
```diff
@@ -5,17 +5,110 @@
...
+  {
+    id: "lh4pj4meWO0",
+    title: "The Tachibanas' Ochugen Season EP 113 | Atashin'chi | [ENG sub]",
+    creator: "[Anime] Atashin'chi Official Channel",
+    about: `The Tachibanas are the quintessential Japanese family, unassuming and infinitely relatable.
+  There’s Mother, who isn’t the best cook and is quite lazy, and Father, who always leaves the bathroom door ajar. The high school daughter Mikan seems a bit of a buffoon, and the middle schooler son Yuzuhiko appears to be quite the cold fish at first glance. They’re a bit peculiar… but they were a family everyone could relate to. And they’re back, this time on YouTube!"`,
+    views: "1,252,816 views",
+    uploadedOn: "Sep 9, 2020",
+  },
+  //   {
```
Comment (aurobindoGupta) [inline src/backend/db/videos.js:55]: remove commented code

File: src/index.js [hunk 4]
```diff
@@ -1,14 +1,23 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { BrowserRouter as Router } from "react-router-dom";
 import App from "./App";
 import { makeServer } from "./server";
+import "./index.css";
+import { MenuProvider, VideoProvider } from "hooks";
```
Comment (aurobindoGupta) [inline src/index.js:7]: you can sort the imports to look clean, like all the local imports from different files at the bottom and the imports from dependencies at the top, just nitpicking

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: src/backend/db/videos.js (hunk 2 above)
Comment (surbhi84) [reply src/backend/db/videos.js:8]: it is the id of the video

File: src/App.js (hunk 1 above)
Comment (surbhi84) [reply src/App.js:30]: that is testing route of mockman

**Pull request merged**